tools/harness-automation/cases_R140/med_9_2_10.py

```diff
@@ -33,10 +33,12 @@
 from autothreadharness.harness_case import HarnessCase
 
 
-class ED_9_2_10(HarnessCase):
-    role = HarnessCase.ROLE_ED
+class MED_9_2_10(HarnessCase):
+    role = HarnessCase.ROLE_MED
```

Comment:
is it `ROLE_MED` for both external and internal TH release?

Reply:
yes

Reply:
in TopologyConfig2018_09_07b.txt : 
9.2.10-Leader:OpenThread,Router_1:SiLabs,MED_1:NXP,SED_1:OpenThread,Commissioner:ARM